### ISSUE TITLE
babel-loader dependency issue

### ISSUE THREAD

**Issue by andreycruz16** [alan2207/bulletproof-react#61]
**Windows Env:**
npm: `8.4.1`
node: `17.5.0`

**Steps to reproduce:**

1. `npm install`
2. `npm start`

![image](https://user-images.githubusercontent.com/11815512/155868144-54efa7c7-b457-423b-b99e-3d54c7974c62.png)

**Comment by alan2207** [comment]
I am planning to update the setup to CRA v5 or Vite.js at some point.

Until then, you can try to bypass the error by adding `SKIP_PREFLIGHT_CHECK=TRUE` to your `.env` file as suggested in order to make it work.

**Comment by gjdoyle** [comment]
Is there any update here? I have tried all of the steps in the output. And changing the .ENV does not work for me. I have also tried adding babel-loader and babel-eslint as optional dependencies in the package.json?

**Comment by nirmalathapa** [comment]
Hi! I am also having the same issue. I have setup the SKIP_PREFLIGHT_CHECK=true in .env file however I am getting types error. How can I resolve this issue?
<img width="722" alt="Screenshot 2022-09-24 at 18 13 38" src="https://user-images.githubusercontent.com/1160546/192108261-2a990ae6-7843-453c-84e3-c2da60da3f63.png">

**Comment by ThePiyushAggarwal** [comment]
I wanted to mention that you guys need to use `yarn install` instead of `npm install`

**Comment by SatyaAchanta** [comment]
@andreycruz16 

My two cents on this, I was also facing the same issue, then I see one of the log messages during this process is exactly this  `../../../package.json` having no license field. I was like..... wait whattttt ? if I am running `yarn start` from root dir in this project why app is finding package.json 3 levels above ? then when I looked into my directories, I found I have `node_modules`, `package.json`, `yarn.lock`. I deleted them and performed these steps again.

1. `rm -rf node_modules yarn.lock`
2. `yarn install`
3. `yarn start`

I could overcome the issue. However, I am still unable to run app successfully, its related to another issues on this project, looking into it.
